src/packer.pkr.hcl

```diff
@@ -0,0 +1,134 @@
+variable "ami_regions" {
+  default     = []
+  description = "The list of AWS regions to copy the AMI to once it has been created. Example: [\"us-east-1\"]"
+  type        = list(string)
+}
+
+variable "build_region" {
+  default     = "us-east-1"
+  description = "The region in which to retrieve the base AMI from and build the new AMI."
+  type        = string
+}
+
+variable "build_region_kms" {
+  default     = "alias/cool-amis"
+  description = "The ID or ARN of the KMS key to use for AMI encryption."
+  type        = string
+}
+
+variable "is_prerelease" {
+  default     = false
+  description = "The pre-release status to use for the tags applied to the created AMI."
+  type        = bool
+}
+
+variable "region_kms_keys" {
+  default     = {}
+  description = "A map of regions to copy the created AMI to and the KMS keys to use for encryption in that region. The keys for this map must match the values provided to the aws_regions variable. Example: {\"us-east-1\": \"alias/example-kms\"}"
+  type        = map(string)
+}
+
+variable "release_tag" {
+  default     = ""
+  description = "The GitHub release tag to use for the tags applied to the created AMI."
+  type        = string
+}
+
+variable "release_url" {
+  default     = ""
+  description = "The GitHub release URL to use for the tags applied to the created AMI."
+  type        = string
+}
+
+variable "skip_create_ami" {
+  default     = false
+  description = "Indicate if Packer should not create the AMI."
+  type        = bool
+}
+
+data "amazon-ami" "debian_bullseye" {
+  filters = {
+    name                = "debian-11-amd64-*"
+    root-device-type    = "ebs"
+    virtualization-type = "hvm"
+  }
+  most_recent = true
+  owners      = ["136693071363"]
+  region      = var.build_region
+}
+
+locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+
+source "amazon-ebs" "example" {
+  ami_block_device_mappings {
+    delete_on_termination = true
+    device_name           = "/dev/xvda"
+    encrypted             = true
+    volume_size           = 8
+    volume_type           = "gp3"
+  }
+  ami_name                    = "example-hvm-${local.timestamp}-x86_64-ebs"
```

Comment:
This AMI should probably get a real name at some point real soon.

Reply:
Well the base AMI is clearly wrong as well since it isn't Windows.  I assume that nothing has really been done with this repo other than creating it.

Reply:
I believe that's true- just wanted to mention it since we want to avoid multiple repos creating AMI's named `example-...`.

Reply:
Hey guys, just for context, this repository got created just before Zach left. No work has been done to it since. I'm starting work on it from today moving forward. I'll give it a real name soon. probably `commando-vm-${local.timestamp}-x86_64-ebs`

Reply:
@itsmostafa - Thanks for letting us know.  Per our dev guide, please follow the ["new project setup" steps](https://github.com/cisagov/development-guide/tree/develop/project_setup) and make those changes in a [`first-commits` branch/PR](https://github.com/cisagov/development-guide/tree/develop/project_setup#create-an-initial-pull-request), rather than in this lineage update PR.  If you have any questions, please let us know.